drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1003,7 +1003,7 @@ function DragAndDropBlock(runtime, element, configuration) {
     };
 
     var wrapMessage = function(element, message) {
-        return "<"+element+">"+message+"</"+element+">";
+        return "<" + element + ">" + message + "</" + element + ">";
     };
 
     var concatFeedbackMessages = function(messages, wrapper_element, prefix) {
```

Comment:
@e-kolpakov I still think you can remove this method. When you do, `wrapMessage` will probably become obsolete as well.